src/main/java/com/artipie/rpm/CliArguments.java

```diff
@@ -38,9 +38,11 @@
  * Cli tool argument parsing.
  *
  * @since 0.9
- * @todo #89:30min Cli options are not parsed correctly. getOptionValue method always return
- *  default value for both options, so it's not possible to override them. But the tool is working
- *  correctly. To build it use `cli` maven profile, see the README.
+ * @todo #113:30min Improve CliArgument behavior.
+ *  Currently CliArgument requires the argument value to be informed immediately
+ *  after the argument (i.e. -psha1, -ffalse, -dsha256). It would be more
+ *  readable if the argument values were preceded by a whitespace (-p sha1,
+ *  -f false, -d sha256, etc)
```

Comment:
@paulodamaso since we already know it's going to be impossible to achieve that with commons-cli, is it worth the project energy to bother with it? WDYT @g4s8?

Reply:
@paulodamaso @g4s8 an alternative would be to advocate for using -n=sha1 for example, I think this should work out of the box and we just add an example in the README and it's done.

Reply:
@victornoel Great cacth! This solves a lot, at least for me. I've changed the issue to add tests with `=`, added this option to `CliArguments` javadoc and removed the puzzle about improving parsing